parser/c2_parser.c2

```diff
@@ -189,9 +189,11 @@ fn void Parser.error(Parser* p, const char* format @(printf_format), ...) @(nore
     longjmp(&p.jmpbuf, 1);
 }
 
-fn void Parser.parseModule(Parser* p, bool is_generated) {
+fn void Parser.parseModule(Parser* p, bool is_interface, bool is_generated) {
     p.expectAndConsume(Kind.KW_module);
-    p.expectIdentifier();
+    // accept builtin types as module names
+    if (!(p.tok.kind >= Kind.KW_bool && p.tok.kind < Kind.KW_void))
+        p.expectIdentifier();
 
     const char*modname = p.pool.idx2str(p.tok.name_idx);
```

Comment:
p.tok.name_idx is only valid if the token.kind is Identifier.

Reply:
No, it is also valid for keywords :)  No change needed, I should document this.